### PR TITLE
added info on hubs for note

### PR DESCRIPTION
**Pull request by jc-berger**
see Jira: https://issues.redhat.com/browse/ACM-9312

### CODE REVIEW

**Comment by dockerymick** [comment]
Link to original PR: https://github.com/stolostron/rhacm-docs/pull/6051#issuecomment-1919381699
Dev approval from original PR: https://github.com/stolostron/rhacm-docs/pull/6051#pullrequestreview-1854149706

File: business_continuity/backup_restore/backup_validate.adoc [hunk 1]
```diff
@@ -3,7 +3,9 @@
...
 
-*Note:* A hub cluster is automatically imported and managed by itself using the `local-cluster` managed cluster. If you disable this by setting `disableHubSelfManagement=true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster and the policy templates do not produce any reports.
+*Note:* A hub cluster is automatically imported and managed by itself by using the `local-cluster` managed cluster. If you disable this by setting `disableHubSelfManagement=true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster, and the policy templates do not produce any reports.
+
+If a cluster hub is managed by a global hub cluster, or if it is installed on a managed cluster instance, the `disableHubSelfManagement=true` setting is disabled. In this instance, you can enable the `backup-restore-enabled` policy. Enable the policy by setting the `is-hub=true` label on the `ManagedCluster` resource that represents the managed hub.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_validate.adoc]:
```suggestion
If a cluster hub is managed by a global hub cluster, or if it is installed on a managed cluster instance, the `disableHubSelfManagement=true` setting is disabled. In this instance, you can enable the `backup-restore-enabled` policy. Enable the policy by setting the `is-hub=true` label on the `ManagedCluster` resource that represents the managed hub cluster. 
```

**Review by dockerymick — APPROVED** [review]
small update, otherwise this looks good to me !

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**